detemplatize.js

```diff
@@ -7,11 +7,13 @@ function replace_with_comments(match) {
 }
 
 module.exports = function (text) {
+  // /* eslint-disable */anything/* eslint-enable */ is removed
   // {# comments #} is replaced with a /* ... */
   // {% anything %} is replaced with a /* ... */
   // {% raw xxx %} is replaced with a {/* ... */} -- this is typically assigned to a variable
   // {{ anything }} is replaced with a {/* ... */} -- this is typically assigned to a variable
   return text
+    .replace(/\/\* eslint-disable \*\/[\s\S]*?\/\* eslint-enable \*\//g, '')
```

Comment:
Should we introduce a but more flexibility by

- Making the `/* eslint-enable */` optional, i.e. ignore the rest of the file after `/* eslint-disable */`?
- Allowing arbitrary whitespaces within the comment, e.g. `/*    eslint-disable */`?